database/postgres/pg.js

```diff
@@ -0,0 +1,45 @@
+const { Client } = require('pg')
+require('dotenv').config()
+
+const pgModule = {}
+let client
+
+pgModule.startPGDB = ()=>{
+    client = new Client({
+      host: process.env.HOS,
+      port: process.env.PORT,
+      user: process.env.USERNAME,
+      password: process.env.PASSWORD,
+      database: process.env.DATABASE
+    })
+    return client.connect()
+}
+
+pgModule.closePGDB = ()=>{
+    return client.end()
+}
+
+pgModule.createPgAccount = async (username, password)=>{
+    if(!username || !password) return
```

Comment:
missing test case. if you call `deletePgAccount()`, then `query` should not have been called.